ministryofjustice/correspondence_tool_staff
# Default disclosure flag to true for non-standard FOIs

Defaults to flag for disclosure specialist to true, and then hides these
options from the user when the case is non standard.

## Self-review checklist


### Related JIRA tickets
[CT-2317](https://dsdmoj.atlassian.net/browse/CT-2317)

### Manual testing instructions
1. create case > foi
2. for type select either of the non-standard options, you should see that the "Flag for disclosure specialist" radio option at the bottom of the page should hide itself. If you click standard case type it should show itself again.
3. create the non-standard case. It should have automatically been flagged for a disclosure specialist.
